source/services/new-relic.rst

```diff
@@ -0,0 +1,90 @@
+New Relic
+=========
+
+To enable New Relic monitoring for an environment:
+
+#. Get a license key. This can be found by clicking on 'Account Settings' from the dropdown menu at
```

Comment:
We have a lot of subaccounts these days. Can we add instructions to first switch to the appropriate account/subaccount for the project?